src/patch-accessors.js

```diff
@@ -104,7 +104,8 @@ let OutsideAccessors = {
      * @this {HTMLElement}
      */
     get() {
-      return this.getAttribute('class');
+      let className =  this.getAttribute('class');
+      return this.getAttribute('class') ? className : '';
```

Comment:
Why use `getAttribute` twice?
`return this.getAttribute('class') || ''` should be sufficient.